presto-spi/src/main/java/com/facebook/presto/spi/eventlistener/QueryStatistics.java

```diff
@@ -23,9 +23,13 @@
     private final Duration cpuTime;
     private final Duration retriedCpuTime;
     private final Duration wallTime;
-    private final Duration queuedTime;
     private final Duration waitingForPrerequisitesTime;
+    private final Duration queuedTime;
```

Comment:
If we are to swap the order of `waitingForPrerequisitesTime` and `queuedTime`, we probably would do the same for their getters. Or we don't have to swap the order.

Reply:
Good call. Let's swap, making it more intuitive.